src/webots/sound/WbSoundClip.hpp

```diff
@@ -42,6 +42,10 @@ class WbSoundClip {
   unsigned int mBuffer;
   int mSide;  // 0: both sides, -1: left only, 1: right only
   double mBalance;
+
+private:
+  WbSoundClip(const WbSoundClip &other);
+  WbSoundClip &operator=(const WbSoundClip &other);
```

Comment:
This doesn't seem to be used.

Reply:
Removed as no heap members remain in class that require a copy prevention (was mDevice).